### PR TITLE
Changed script owner

### PR DESCRIPTION
**Pull request by Alium58**
Changed script owner to loathers
Moved from SVN to git for install instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 To install this script in KolMafia, simply use the following command in the gCLI:
 
-`svn checkout https://github.com/RESPRiT/bountiful/branches/release`
+`git  checkout https://github.com/loathers/bountiful.git`
```
Comment (docrostov) [inline README.md]: i believe we can now (starting with r26691) use `git checkout loathers/bountiful main` but i am not sure, we should test this. and i am open to the idea that we should (for backwards compatibility reasons) just use the URL

File: README.md [hunk 2]
```diff
@@ -47,6 +47,6 @@ This script is still in the "beta" stages and has been tested but not thoroughly
 
 The code is documented using JavaDoc/JSDoc style comments if you are interested in taking a peek at the source code.
 
-This script is adapted (but very different) from AutoBHH, originally by izchak and majorly revised by Raorn and Zarqon.
+This script is forked from RESPRiT. Which was adapted (but very different) from AutoBHH, originally by izchak and majorly revised by Raorn and Zarqon. Thank you all for your contributions to this script.
```
Comment (docrostov) [inline README.md]: we should link the original repo. once we merge this i am going to disconnect the fork so this in the readme will be the connection to the O.G. script

File: scripts/bountiful.ash [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-* -Major revisions by Raorn and Zarqon
+* Forked from RESPRiT
+* -Which was adapted from AutoBHH:
+*  -Originally by izchak
+*  -Major revisions by Raorn and Zarqon
 ******************************************************************************/
 script "Bountiful";
-notify tamedtheturtle;
-since r18000;
+since r26686;
```
Comment (docrostov) [inline scripts/bountiful.ash]: let's do r26691 for this, since that's when gausie added the support for git install w/o the full URL

**Review by docrostov — APPROVED** [review]
looks good, if you could just swap out the r26691 comment and test the string i gave you re: a new install change that would be great. ping me on discord when ready to merge :-)

File: README.md (hunk 1 above)
Comment (Alium58) [reply README.md]: Good call, I like the simpler approach. Believe we don't need to include `main` at the end either, it is the default. I'll test it out

Comment (Alium58) [reply README.md]: Yep `git checkout loathers/bountiful` works great

**Pull request merged**